go-chaos/internal/k8helper.go

```diff
@@ -70,14 +70,23 @@ func createK8Client(kubeconfig *string) (K8Client, error) {
 	return K8Client{Clientset: clientset, ClientConfig: clientConfig}, nil
 }
 
-func findKubeconfigPath() *string {
+type KubernetesSettings struct {
+	kubeConfigPath string
+	namespace      string
+}
+
+func findKubernetesSettings() KubernetesSettings {
 	//// based on https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go
 	var kubeconfig *string
 	if home := homedir.HomeDir(); home != "" {
 		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
 	} else {
 		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
 	}
+	namespace := flag.String("namespace", "", "Kubernetes namespace to use")
```

Comment:
:x: This is a bit hidden. I would like to use a presistent flag https://stackoverflow.com/a/63498490/2165134 as we do with verbose here https://github.com/zeebe-io/zeebe-chaos/blob/main/go-chaos/cmd/root.go#L53

Reply:
Yeah, that's what I tried first but that introduced a circular dependency from cmd -> internal -> cmd again. I guess that's why the kubeconfig parameter is also defined here and not in cmd?

Reply:
well what you need to do is then to set a internal property like we do for verbosity see here https://github.com/zeebe-io/zeebe-chaos/blob/main/go-chaos/cmd/root.go#L48

Reply:
Ahh, neat :+1: Do you remember if there was a reason to not do the same for the kubeconfig param?